### PR TITLE
Update README.md to make the difference of CLIs clearer

### PR DESCRIPTION
**Pull request by rberrelleza**
# Proposed changes
Update README.md to make the difference between Open Source CLI and Platform CLI clearer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ The Okteto CLI has two operational modes:
...
 
-Okteto Open Source supports the following commands:
+The Okteto Open Source CLI supports the following commands:
 
 - `okteto context`
 - `okteto up`
 - `okteto down`
 
-For reference, our Open Source CLI supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional) of the Okteto Manifest.
+> It's important to note that the Okteto Open Source CLI only supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional) of the Okteto Manifest.
```
Comment (rberrelleza) [inline README.md:56]: @codyjlandstrom I find it confusing that we are sending people that are using the Okteto CLI in pure cluster mode to https://okteto.com/docs for reference. Do we have an in-repo description of the manifest? would that make sense?

**Review by codyjlandstrom — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (codyjlandstrom) [reply README.md:56]: Maybe we could have a simplified one? I fear that the full dev section of the manifest is too long to place here and keep maintained

File: README.md [hunk 2]
```diff
@@ -45,15 +45,17 @@ The Okteto CLI has two operational modes:
...
-Okteto Open Source supports the following commands:
+The Okteto Open Source CLI supports the following commands:
 
 - `okteto context`
 - `okteto up`
 - `okteto down`
 
-For reference, our Open Source CLI supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional) of the Okteto Manifest.
+> [!NOTE]
+ ⚠️ Notice: The open-source version of Okteto only supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional)  of the Okteto manifest. For additional features and full functionality, consider exploring the [Okteto Platform](https://www.okteto.com/get-demo/).
```
Comment (andreafalzetti) [inline README.md:57]:
```suggestion
 ⚠️ Notice: The open-source version of Okteto only supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional) of the Okteto manifest. For additional features and full functionality, consider exploring the [Okteto Platform](https://www.okteto.com/get-demo/).
```

Comment (andreafalzetti) [reply README.md:57]: extra space

**Review by andreafalzetti — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rberrelleza) [reply README.md]: yeah, that's what I was thinking as well. it can help reinforce the difference. but that's out o the scope of this PR.

**Pull request merged**